content/api-streamer/producer.textile

```diff
@@ -17,8 +17,6 @@ languages:
 
 The "Ably API Streamer":https://ably.com/api-streamer makes it easy and cost-effective for organizations and developers to offer realtime APIs, at any scale. It provides a unified solution for publishing, managing and distributing realtime data via open protocol APIs, all built on the global "Ably Platform":https://ably.com/platform. API Streamer, amongst other things, is being used to share live sports scores and financial market data, mobility IoT data such as GPS locations, and SaaS integration updates in real time.
 
-Our API Streamer docs are currently in beta. While we are working hard to improve them, please feel free to "reach us out":https://ably.com/support in case of any queries.
```

Comment:
"reach us out"?

Reply:
Yes, but I think I've deleted all paragraphs that refer to "reach us out" and "beta docs". Let me know if you were referring to one I've missed though!

Reply:
Sorry @m-hulbert , that was supposed to be an approving review with a comment (of the "really?" variety) rather than a request for changes. Approving now!